lib/Net/SAML2/Binding/SOAP.pm

```diff
@@ -176,11 +198,13 @@ sub handle_request {
 
     my $saml = _get_saml_from_soap($request);
     if (defined $saml) {
-        $self->verify_xml(
-            $saml,
-            cert_text => $self->idp_cert,
-            cacert    => $self->cacert
-        );
+        foreach my $cert (@{$self->idp_cert}) {
```

Comment:
Same here.

Reply:
yeah, eval I think and some plan for what errors to return if multiple (all) certs failed